src/protocol/securemul.rs

```diff
@@ -108,11 +103,9 @@ pub mod tests {
     #[allow(clippy::cast_possible_truncation)]
     pub async fn concurrent_mul() {
         type MulArgs<F> = (Replicated<F>, Replicated<F>);
-        async fn mul<F: Field>(
-            v: (ProtocolContext<'_, Arc<InMemoryEndpoint>, F>, MulArgs<F>),
-        ) -> Replicated<F> {
+        async fn mul<F: Field>(v: (ProtocolContext<'_, F>, MulArgs<F>)) -> Replicated<F> {
```

Comment:
thank you for removing the `Arc<InMemoryEndpoint>`... it scares me to see that every time =).